src/components/Navbar.js

```diff
@@ -1,15 +1,41 @@
-import React from "react"
-import { Link } from "react-router-dom"
+import Parse from "parse"
+import { useState } from "react"
+import Cat from "../images/cat.jpg"
 
-const Navbar = () => {
+export default function Navbar() {
+
+    const [currentUser] = useState(Parse.User.current().getUsername())
```

Comment:
I suspect you don't really need the useState here. You can just make a const.
`const currentUser = Parse.User.current().getUsername()`